invenio_search/ext.py

```diff
@@ -405,6 +405,20 @@ def _build(tree_or_filename, alias=None):
                     name=action["alias"],
                     ignore=ignore,
                 )
+    def update_mapping(self, index):
+        """Update mapping of the existing index."""
+        mapping_path = self.mappings[index]
```

Comment:
comment/question: just to confirm, this means that you have to call this with the full index name, i.e. `rdmrecords-records-record-v5.0.0`? I think it's good for now, since we're explicitly using it in upgrades.

Reply:
yes, see here: https://github.com/inveniosoftware/docs-invenio-rdm/pull/524/files#diff-847bf690b840996584043335af5cf62e8939572e6d086cba94fd3eecaf377966R85